packages/mds-event-processor/src/proc.ts

```diff
@@ -0,0 +1,81 @@
+import db from '@mds-core/mds-db'
+import cache from '@mds-core/mds-cache'
+import http from 'http'
+import log from '@mds-core/mds-logger'
+
+const env = process.env
+
+async function resetAll(type: string) {
+  // cache related
+  if (type === 'event') {
+    await cache.delCache('device:state')
+  } else if (type === 'trip') {
+    await cache.delCache('trip:state')
+    await cache.delCache('trips:events')
+    await cache.delCache('trips:telemetry')
+  } else if (type === 'trip') {
+    await cache.delCache('provider:state')
+  }
+
+  // database related
+  await db.initialize()
+  if (type === 'event') {
+    await db.resetTable('reports_device_states')
+  } else if (type === 'trip') {
+    await db.resetTable('reports_trips')
+  } else if (type === 'provider') {
+    await db.resetTable('reports_providers')
+  }
+}
+
+async function dataHandler(
```

Comment:
I _really_ think we should just use express for this, that would cut down on the complexity here a lot.

Reply:
Michael and I chatted, and think it would make sense to even refactor this into an MDS-Knative-Layer for use across any packages that consume knative events (could be used in here and the provider-processor). Wanna take this up @macsj200?

Reply:
Last I heard @drtyh2o is taking the helm on this, see https://lacunadotai.atlassian.net/browse/MDSAAS-444